m10_point_of_interest/01_introduction.rst

```diff
@@ -0,0 +1,61 @@
+Point Of Interest (POI)
+=======================
+
+.. styled-topic::
+
+  Main questions:
+      What are point of interest directives, why and how to use it.
+
+  Topics?
+      What, why and how.
+
+  Difficulty:
+      Easy
+
+  Laboriousness:
+      Around 5 minutes of reading
+
+
+Introduction to the Point of Interest directive.
+
+
+What is a point of interest?
+----------------------------
+
+Directive for creating a "point of interest" summary block.
+
+A point of interest is similar to a normal admonition ("coloured info box"), but
+the point of interest are also linked to each other with next/previous links. The links enable
+the user to quickly navigate between different points of interest.
+
+Point of interests may also be used to generate separate lecture slides
+(not directly included in the A+ content chapters). This requires a separate
+tool called :doc:`Presentation Maker </m10_point_of_interest/03_presentation_maker>`
+
+.. point-of-interest:: This is a point of interest
+
+  And here is the content. More examples in the :doc:`next chapter </m10_point_of_interest/02_examples>`
+
+From now on I will refer to point of interest as a POI.
+
+Why to use it?
+--------------
+
+#. Highlight important parts and make them easily accessible with links.
+
+#. Turn your Point of interest into presentation slides with Presentation Maker.
+
+How to use POI?
+---------------
+
+1. Point of Interest should work out of the box, but if you ran into problems:
+
+    - Try updating your a-plus-rst-tools. `Read this, especially the 3rd point. <https://github.com/apluslms/course-templates#installation>`_
+        - Updating should work out with this: :code:`git submodule update`
```

Comment:
`git submodule update` alone does not update the submodule to the latest available commit. It sets the submodule to the commit that has been set in the main repo for the submodule. However, the a-plus-rst-tools README has instructions about upgrading the rst-tools and you could link to it:
https://github.com/Aalto-LeTech/a-plus-rst-tools/blob/master/README.md#upgrading-the-tools